libs/pages/layout/src/lib/feature/layout/layout.spec.tsx

```diff
@@ -1,5 +1,6 @@
 import { render } from '__tests__/utils/setup-jest'
```

Comment:
You need to use `renderWithProviders`

Reply:
https://github.com/Qovery/console/pull/1265/commits/b14fb5c29eefc7764c12518cb4a6bffc7dfd5347